api/trace/api.go

```diff
@@ -60,6 +60,30 @@ func WithEndTime(t time.Time) EndOption {
 	}
 }
 
+// ErrorConfig provides options to set properties of an event at the time
+// that an error is recorded.
```

Comment:
This reads a bit weird. How about:

```go
// ErrorConfig provides options for setting properties of an error
// event at the time of recording it.
```

Reply:
I updated the docblocks to more consistently refer to error events.  Let me know what you think.